types/microsoft-graph/index.d.ts

```diff
@@ -9590,7 +9675,7 @@ export interface Subscription extends Entity {
     includeResourceData?: NullableOption<boolean>;
     /**
      * Optional. Specifies the latest version of Transport Layer Security (TLS) that the notification endpoint, specified by
-     * notificationUrl, supports. The possible values are: v1_0, v1_1, v1_2, v1_3. For subscribers whose notification endpoint
+     * notificationUrl, supports. The possible values are: v10, v11, v12, v13. For subscribers whose notification endpoint
```

Comment:
These values seem wrong, can we validate that they have changed in this way? This feels like it could be a breaking change for many consumers if this is correct.